### PR TITLE
Delete VPA pointer from PodState

### PR DESCRIPTION
**Pull request by tkulczynski**

### CODE REVIEW

File: vertical-pod-autoscaler/recommender/model/cluster_test.go [hunk 1]
```diff
@@ -139,39 +139,44 @@ func TestUpdatePodSelector(t *testing.T) {
...
 	vpa = cluster.Vpas[testVpaID]
 	assert.Equal(t, pod, vpa.Pods[testPodID])
-	assert.Equal(t, vpa, pod.Vpa)
+	assert.Equal(t, map[VpaID]*Vpa{testVpaID: vpa}, pod.MatchingVpas)
 }
 
 // Creates a VPA and a matching pod, then add another VPA matching the same pod.
-// Verifies that the pod is controlled by the first VPA.
-// Next deletes the first VPA and verfies that the pod is controlled by the
+// Verifies that the pod is knows about both of them.
```
Comment (bskiba) [inline vertical-pod-autoscaler/recommender/model/cluster_test.go]: Remove is

Comment (tkulczynski) [reply vertical-pod-autoscaler/recommender/model/cluster_test.go]: Done

**Comment by bskiba** [comment]
I think the aws test is flaking (filed https://github.com/kubernetes/autoscaler/issues/608)

**Pull request merged**